website/docs/r/maintenance_window.html.markdown

```diff
@@ -43,10 +45,10 @@ Once maintenance is scheduled for your cluster, you cannot change your maintenan
 * `project_id` - The unique identifier of the project for the Maintenance Window.
 * `day_of_week` - Day of the week when you would like the maintenance window to start as a 1-based integer: S=1, M=2, T=3, W=4, T=5, F=6, S=7.
 * `hour_of_day` - Hour of the day when you would like the maintenance window to start. This parameter uses the 24-hour clock, where midnight is 0, noon is 12 (Time zone is UTC).
-* `start_asap` - Flag indicating whether project maintenance has been directed to start immediately. If you request that maintenance begin immediately, this field returns true from the time the request was made until the time the maintenance event completes.
+* `start_asap` - Boolean flag that indicates whether MongoDB Atlas starts the maintenance window immediately upon receiving this request. To start the maintenance window immediately for your project, MongoDB Atlas must have maintenance scheduled and you must set a maintenance window. This flag resets to false after MongoDB Atlas completes maintenance.
 * `number_of_deferrals` - Number of times the current maintenance event for this project has been deferred, you can set a maximum of 2 deferrals.
-* `defer` - Defer the next scheduled maintenance for the given project for one week.
-* `auto_defer` - Defer any scheduled maintenance for the given project for one week.
+* `defer` - Defers the maintenance window for the specified project for one week.
+* `auto_defer` - Toggles automatic deferral of the maintenance window for the specified project for one week.
```

Comment:
question: are these two also booleans?

Reply:
@jwilliams-mongo confirmed. see source code here to these two parameters for more details: https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/mongodbatlas/resource_mongodbatlas_maintenance_window.go#L72

Reply:
also adjusted slightly to reflect this. feel free to edit as you see fit